Update documentation with Spec file not in upstream (#36)
packit-service/packit.dev: content/packit-as-a-service/_index.md

For more info, see
[.packit.yaml documentation](/docs/configuration/).

### Add RPM spec file

Packit Service needs the RPM spec file to build your project.

> Either you have to include it into your upstream repository or

Reviewer: ```suggestion
Either include it in your upstream repository or
```

> if you do not want to have it in your upstream repository, for some reason,

Reviewer: ```suggestion
if you do not want to have it in your upstream repository,
```
